cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -9,22 +9,26 @@ toc::[]
 [id="odo-notable-improvements_{context}"]
 == Notable changes and improvements in `{odo-title}`
 
-* `{odo-title}` integration with devfiles is no longer a Technology Preview feature and is now fully supported. To learn more, see xref:../../cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/using-devfiles-in-odo.adoc#creating-a-java-application-by-using-a-devfile[Creating applications by using devfiles].
+* `{odo-title}` now supports Devfile v2.
 
-* The default deployment mechanism in `{odo-title}` is the devfile deployment now.
+* `odo create -s2i` now converts an S2I component into a devfile component. When running `odo create --s2i <component-type>` `{odo-title}` now creates a converted Devfile component based on the S2I images of the specified component type.
++
+Note that this feature introduces many breaking changes, see xref:../../cli_reference/developer_cli_odo/odo-release-notes.adoc#odo-known-issues_odo-release-notes[Known Issues] to learn more.
+
+* Operator based service is now created on the cluster only after you run `odo push` and not after `odo service create` anymore.
 
-* Added documentation about converting S2I to devfile. See xref:../../cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/using-devfiles-in-odo.adoc#converting-an-s2i-component-into-a-devfile-component_creating-applications-by-using-devfiles[Converting an S2I component into a devfile component].
+* The `--container` flag now allows you to specify to which container to attach storage when running `odo storage create` command. See xref:../../cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc#adding-storage-to-a-specific-container_working-with-storage[Adding storage to a specific container] to learn the details.
```

Comment:
IBM SG `allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."` - However

Reply:
If you prefer to keep the feature as the subject, you can usually replace "allows" with "enables".